src/oppgaver/oppgave4/contexts/DarkMode.js

```diff
@@ -0,0 +1,14 @@
+import React, { createContext, useContext, useState } from "react";
+
+// 1. Lag DarkMode slik du gjorde i oppgave 3
```

Comment:
Jeg tror kanskje ikke det er noe vits for dem å lage denne på nytt. Synes vi bare skal copy paste inn ferdig-laga DarkMode-context fra oppgave 3.